enterprise/backend/src/metabase_enterprise/serialization/v2/extract.clj

```diff
@@ -46,9 +46,16 @@
   [opts]
   (log/tracef "Extracting Metabase with options: %s" (pr-str opts))
   (serdes.backfill/backfill-ids)
-  (let [model-pred (if (:data-model-only opts)
+  ;; TODO document and test data-model-only if we want to keep this feature...
+  (let [model-pred (cond
```

Comment:
It's surprising that `:data-model-only` and `:include-field-values` are exclusive of each other. I think I would prefer requiring both options to get data-model-only with `FieldValues`. 

If not the interaction should, at least, be documented and tested.

Reply:
I'll work out the details with @luizarakaki who, I'm sure, already told me about how this is supposed to work 🤔 unfortunately, as the TODO states, `:data-model-only` is not tested or documented.

I opened a PR to fix this, if that's the right thing to do.